Lib/pkgutil.py

```diff
@@ -119,6 +119,9 @@ def iter_modules(path=None, prefix=''):
     """
     if path is None:
         importers = iter_importers()
+    elif isinstance(path, str) or isinstance(path, bytes):
```

Comment:
bytes, bytearray, array('b') (or whatever the right incantation is), memoryview....